### ISSUE TITLE
Improving accessibility access for scene

### ISSUE THREAD

**Issue by superjai** [ioos/sanctuarywatch#13]
So, let's do some things to improve accessibility for our site. The first thing to do is to understand the gold-standard principles for site accessibility. The acronym for site accessibility here is POUR (or the POUR principles).

- Read through [this article](https://webaim.org/articles/pour/) to learn about the POUR principles (please be sure to read through all five pages).

The O in POUR stands for operable and let's increase the operability of the scene for those with accessibility issues. A key way to do this is to use [semantic HTML](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML), meaning that we are using HTML elements for their intended purpose - which makes it easier for things like screen readers to make sense of what is going on, on a page (please do have a read of the linked article for a more in depth description of semantic HTML).

One place where our HTML could be improved is the links in the table of contents within a scene. If one can't use a mouse and is using a keyboard only (a big consideration for the O in POUR), you should be able to get to all links by use of the tab key. You can't get to table of contents links via the tab key at the moment, but that can be easily fixed. All you need to do is to add the following within the `<a>` tag of all table of content entries

- `href = "#"`

Let's also add a bit to the stylesheet of the page, so that - if a user is using the keyboard only to navigate the page - they can see where the tab is currently focused.
```
a:focus, button:focus {
  outline: 2px solid blue;
}
```

You may need to adjust this css, so that each "tab-able" element is spotlighted on tab focus. Also, outlining the border with a solid blue line may not be the best way to show focus. Experiment a bit and see if there is something else that looks better. 

Please go ahead and implement this and double check that doing so makes all links in the scene available via the tab and return keys. We'll do more for accessibility (modal windows are notoriously tricky in this regard), but this is an easy first step.

**Comment by skanda-vasishta** [comment]
on focus on modal icon in table of contents, highlight associated icon in svg image

**Comment by skanda-vasishta** [comment]
styling implemented here and below: https://github.com/ioos/sanctuarywatch/blob/ebf79dd83deb873765f7d05c125001023a90c99f/themes/Sanctuary_Watch/style.css#L535
